vsc/baseline/localization.py

```diff
@@ -93,3 +94,60 @@ def score(self, candidate: CandidatePair, match: Match, box, similarity) -> floa
 class VCSLLocalizationCandidateScore(VCSLLocalization):
     def score(self, candidate: CandidatePair, match: Match, box, similarity) -> float:
         return candidate.score
+
+
+class VCSLLocalizationDnS(VCSLLocalizationMaxSim):
```

Comment:
If this is DnS specific, it might make sense to put it with DnS code -- maybe in dns_baseline? Or as a module within baseline/dns?

Reply:
This is indeed quite DnS specific method. I will move it to dns_baseline, since the baseline/dns will be removed if we go with the torchscript solution for DnS.